### PR TITLE
examples/rawkv: add example for rawkv

### PR DESCRIPTION
**Pull request by AndreMouche**
add example for rawkv

### CODE REVIEW

File: examples/rawkv/rawkv.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 PingCAP, Inc.
```
Comment (Xuanwo) [inline examples/rawkv/rawkv.go:1]: Code should belong to tikv (aka CNCF) instead.

Comment (disksing) [reply examples/rawkv/rawkv.go]: Copyright 2021 TiKV Authors

File: examples/rawkv/rawkv.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 PingCAP, Inc.
```
Comment (disksing) [inline examples/rawkv/rawkv.go:1]: PingCAP, Inc.  -> TiKV Authors

File: examples/rawkv/rawkv.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+import (
+	"fmt"
+
+	"github.com/tikv/client-go/v2/config"
+	"github.com/tikv/client-go/v2/tikv"
+)
+
+func main() {
+	cli, err := tikv.NewRawKVClient([]string{"172.16.6.153:2379"}, config.DefaultConfig().Security)
```
Comment (disksing) [inline examples/rawkv/rawkv.go:24]: better to use some IP like 127.0.0.1

**Review by disksing — APPROVED** [review]

**Pull request merged**